ui/src/forms/index.ts

```diff
@@ -1,15 +1,43 @@
 import { renderer, components, editors } from '@hydrofoil/shaperone-wc/configure'
+import { html } from 'lit-element'
 import { DefaultStrategy } from '@hydrofoil/shaperone-wc/renderer/DefaultStrategy'
-import * as MaterialRenderStrategy from '@hydrofoil/shaperone-wc-material/renderer'
```

Comment:
Guess this would no longer be needed as dependency

Reply:
Good catch